File: April-18/24-04-18/python/02-Noddy-and-his-Vowel.py
def check(string):
    l = len(string)
    a = False
    e = False
    i = False
    o = False
    u = False
    
    for x in range(0,l):
        if(string[x] == 'a'):
            a = True
        if(string[x] == 'e'):
            e = True
        if(string[x] == 'i'):
            i = True
        if(string[x] == 'o'):
            o = True
        if(string[x] == 'u'):
            u = True
    
    if(a and e and i and o and u):
        return True
    else:
        return False

for _ in range(int(input())):
    string = input()
    if(check(string) == True):
        print("YES")
    else:
        print("NO")
    

File: April-18/24-04-18/java/02-Noddy-and-his-Vowel.java

import java.io.*;

class TestClass {
    
    static boolean check(String str)
    {
        int n = str.length();
        boolean a,e,i,o,u;
        a = e = i = o = u = false;
        for(int x = 0; x < n; x++)
        {
            if(str.charAt(x) == 'a')
                a = true;
            if(str.charAt(x) == 'e')
                e = true;
            if(str.charAt(x) == 'i')
                i = true;
            if(str.charAt(x) == 'o')
                o = true;
            if(str.charAt(x) == 'u')
                u = true;
        }
        if(a && e && i && o && u)
            return true;
        else
            return false;
    }
    public static void main(String args[] ) throws Exception {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(input.readLine());
        while(t-- > 0)
        {
            String str = input.readLine();
            if(check(str) == true)
                System.out.println("YES");
            else
                System.out.println("NO");
            
        }
    }
}
